README.md

```diff
@@ -65,6 +65,7 @@ The following table lists the software versions NGINX Gateway Fabric supports.
 | NGINX Gateway Fabric | Gateway API | Kubernetes | NGINX OSS |
 |----------------------|-------------|------------|-----------|
 | Edge                 | 1.0.0       | 1.23+      | 1.25.3    |
+| 1.1.0                | 1.0.0       | 1.23+      | 1.25.3    |
```

Comment:
Elsewhere it seems as though 1.1.0 is "replacing" edge, so this table is confusing to me - should the edge row be removed entirely, or to reflect top down chronological order, should the edge row be second instead of first?

Reply:
Good point. The README in `main` should contain `edge` but I don't think it makes sense in the release branch since we will not update the dependencies in the table if they change for `edge`. 

@sjberman @pleshakov are you guys OK if I remove the edge entry from this table?